examples/pennylane/7_Pulse_control_OQC_Lucy/7_Pulse_control_OQC_Lucy.ipynb

```diff
@@ -0,0 +1,753 @@
+{
```

Comment:
<p>We do not have "about the author" sections in the example notebooks.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/amazon-braket/amazon-braket-examples/pull/448/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='25'/>

Reply:
<p>removed</p>